src/matrix.jl

```diff
@@ -168,6 +168,38 @@ function Base.:(*)(A::T, B::T) where {T<:Matrices}
     LinearAlgebra.mul!(C, A, B)
 end
 
+# scalar multiplication
+function Base.:(*)(c::ArbLike, A::T) where {T <: Matrices}
+    C = T(size(A,1),size(A,2),prec = precision(A))
+    Arblib.mul!(C,A,c)
+end
+function Base.:(*)(c::AcbLike, A::AcbMatLike)
+    C = similar(A)
+    Arblib.mul!(C,A,c)
+end
+function Base.:(*)(c::AcbLike, A::ArbMatrixLike)
+    C = AcbMatrix(A)
+    Arblib.mul!(C,AcbMatrix(A),c)
+end
+Base.:(*)(A::T, c::AcbLike) = c*A
```

Comment:
`where T<:??`

Reply:
Good point, changed that in the last commit